src/generate-oas.ts

```diff
@@ -46,8 +46,8 @@ function appendVersionIfV2(name: string, version: string): string {
  * Helper function. Also contains explicit workaround for some API names.
  */
 export function resolveApiName(name: string, version: string): string {
-  if (name === "Shopper Baskets OAS") {
-    return version === "v2" ? "ShopperBasketsV2" : "ShopperBasketsV1";
+  if (name === "Shopper Baskets OAS" && version === "v2") {
+    return "ShopperBasketsV2";
```

Comment:
Based on this, I assume we've decided to name shopper baskets V1 as `ShopperBaskets` and not `ShopperBasketsV1`?

Reply:
Yes that's correct